includes/core/classes/class-event-setup.php

```diff
@@ -137,13 +139,38 @@ public function register_post_type(): void {
 					'custom-fields',
 				),
 				'menu_icon'     => 'dashicons-nametag',
+				'has_archive'   => true,
 				'rewrite'       => array(
-					'slug' => _x( 'event', 'Post Type Slug', 'gatherpress' ),
+					'slug'       => $rewrite_slug,
+					'with_front' => false,
 				),
 			)
 		);
 	}
 
+	/**
+	 * Returns the post type slug localised for the site langugage and sanitized as URL part.
+	 *
+	 * Do not use this directly, use get_value( 'general', 'urls', 'events' ) instead.
+	 *
+	 * This method switches to the sites default language and gets the translation of 'events' for the loaded locale.
+	 * After that, the method sanitizes the string to be safely used within an URL,
+	 * by removing accents, replacing special characters and replacing whitespace with dashes.
+	 *
+	 * @since 0.31.0
```

Comment:
I'm still leaving all these `1.0.0`.

Reply:
No problemo, reverted.

But...

While testing the [GatherPress code reference](https://playground.wordpress.net/?mode=seamless&blueprint-url=https://raw.githubusercontent.com/carstingaxion/gatherpress-devhub/WIP/use-build-deps/blueprint-import.json) I found it to be not nice to have everything related to a non existent 1.0 release. Because even now, during the initial development in can be helpful to see a list of changed functions, like wp does for each release. This list is technically working for GatherPress too, but it would show the same list (of all functions) for each release. 

But for sure, changing all existing since tags to make the code-reference work needs to be its own issue.